### ISSUE TITLE
`Duplicate Entry` error when building a new uno app on WinUI

### ISSUE THREAD

**Issue by ramezgerges** [unoplatform/uno#17189]
### Current behavior

This only happens with the windows target framework. net8-desktop works fine.

MSBuild version 17.9.8+610b4d3b5 for .NET
  Determining projects to restore...
  Restored D:\myunoapp2\myunoapp2\myunoapp2.csproj (in 844 ms).
  myunoapp2 -> D:\myunoapp2\myunoapp2\bin\Debug\net8.0-windows10.0.19041\myunoapp2.dll
WINAPPSDKGENERATEPROJECTPRIFILE : error : PRI175: 0x80073b0f - Processing Resources failed with error: Duplicate Entry. [D:\myunoapp2\myunoapp2\myunoapp2.csproj::TargetFramework=net8.0-windows10.0.19041]
WINAPPSDKGENERATEPROJECTPRIFILE : error : PRI277: 0x80073b0f - Conflicting values for resource 'Files/Uno.UI.Toolkit/Themes/Generic.xbf' [D:\myunoapp2\myunoapp2\myunoapp2.csproj::TargetFramework=net8.0-windows10.0.19041]

Build FAILED.

WINAPPSDKGENERATEPROJECTPRIFILE : error : PRI175: 0x80073b0f - Processing Resources failed with error: Duplicate Entry. [D:\myunoapp2\myunoapp2\myunoapp2.csproj::TargetFramework=net8.0-windows10.0.19041]
WINAPPSDKGENERATEPROJECTPRIFILE : error : PRI277: 0x80073b0f - Conflicting values for resource 'Files/Uno.UI.Toolkit/Themes/Generic.xbf' [D:\myunoapp2\myunoapp2\myunoapp2.csproj::TargetFramework=net8.0-windows10.0.19041]
    0 Warning(s)
    2 Error(s)

Time Elapsed 00:00:14.43

Workload updates are available. Run `dotnet workload list` for more information.

### How to reproduce it (as minimally and precisely as possible)

```powershell
dotnet new unoapp -o myunoapp2 -preset "recommended" -platforms "windows" -platforms "desktop" -theme-service  -vscode False -pwa False -theme "material" -presentation "mvvm" -config  -di  -log "default" -nav "regions" -http  -loc  -tests "none" -toolkit  -dsp  -id "com.companyname.myunoapp2" -pub "O=myunoapp2"
dotnet build
```

### Works on UWP/WinUI

None

**Comment by jeromelaban** [comment]
Can you attach a binlog?

**Comment by Youssef1313** [comment]
@ramezgerges Also check if it works in VS instead of command-line. I think WinUI didn't support building with `dotnet build`? Though the error used to be something different.

**Comment by jeromelaban** [comment]
Yes, WinUI does not support `dotnet build` well. `msbuild` is best. We still have to get back to the WinUI team on those issues.

**Comment by jeromelaban** [comment]
This does not repro in Uno templates 5.3.5.

@ramezgerges could you try with those templates?

**Comment by assassin316** [comment]
I've started receiving this as well in my project. I did some trial and error and the issue started occurring with these nuget updates:

If I leave my packages at 5.2.175 all compiles well. This happened to me when I migrated to a Single Project strucutre, thought it was something I did during migration, but I then went back to my old structure and as you can see below I was able to reproduce it.

@ramezgerges  can you determine if you get the same behavior when updating those specific packages ?

![image](https://github.com/user-attachments/assets/b607cce9-1993-4416-a756-98dc8b2f5b29)

**Comment by assassin316** [comment]
@jeromelaban  I don't think we can mark this as Closed just yet  😐

**Comment by Youssef1313** [comment]
There should be a new version soon. cc @carldebilly to track the deployment process.

**Comment by Youssef1313** [comment]
@assassin316 I new version of Uno.Sdk is now available 5.4.0-dev.220.

**Comment by assassin316** [comment]
@Youssef1313  that did the trick, solution compiles successfully. Thanks.
